src/components/common/Tab.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+    <nav class="ml-12 -mb-px flex justify-items-center" aria-label="Tabs">
+        <!-- on -->
+        <a
+            href="#"
+            class="border-gray-200 dark:border-darkGray-600 border rounded-t-md"
+            aria-current="page"
+        >
+            <span
+                class="block bg-gray-50 dark:bg-darkGray-900 -mb-px whitespace-nowrap py-3 sm:py-5 px-8 text-blue-900 dark:text-darkGray-300 font-medium rounded-t-md border-gray-50 dark:border-darkGray-900 border-b"
+            >
+                aaa
+            </span>
+        </a>
+
+        <!-- off -->
+        <a
+            href="#"
+            class="border-gray-50 dark:border-darkGray-900 border rounded-t-md"
+            v-for="num in labelsNumArray"
+            :key="num"
+        >
+            <span
+                class="block bg-gray-50 dark:bg-darkGray-900 -mb-px whitespace-nowrap py-3 sm:py-5 px-8 text-blue-500 dark:text-blue-400 font-medium rounded-t-md border-gray-200 dark:border-darkGray-600 border-b hover:text-blue-400 dark:hover:text-blue-300"
+            >
+                {{ labels[num] }}
+            </span>
+        </a>
+    </nav>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent } from 'vue';
+
+export default defineComponent({
+    props: {
+        labels: { type: Array, required: true },
```

Comment:
I recommend using statically-typed properties instead of a generic one for consistency.
Reference: https://v3.vuejs.org/guide/typescript-support.html#annotating-props